lib/routes/account.js

```diff
@@ -2169,6 +2170,22 @@ module.exports = (
             })
         }
 
+        function deleteSecondaryEmailIfUnverified() {
+          return db.getSecondaryEmail(email)
+            .then((secondaryEmailRecord) => {
+              // Only delete secondary email if it is unverified and does not belong
+              // to the current user.
+              if (! secondaryEmailRecord.isVerified && secondaryEmailRecord.uid !== uid.toString('hex')) {
```

Comment:
Heh yea. Turns out that the response from the auth-db gets [unbuffer](https://github.com/mozilla/fxa-auth-db-mysql/blob/8f5653cfb4aa9c75da124f678d4313e006f99684/fxa-auth-db-server/index.js#L224) when sent over and then auth-server [buffers](https://github.com/mozilla/fxa-auth-server/blob/master/lib/db.js#L221) it back. Just needed to add bufferize call. Mind another r?